rent-a-car/src/main/java/xws/tim16/rentacar/service/CustomUserDetailsService.java

```diff
@@ -99,7 +101,7 @@ public void changePassword(String oldPassword, String newPassword) {
                 .address(userDTO.getAddress())
                 .password(passwordEncoder.encode(userDTO.getPassword()))
                 .isAdmin(false)
-                .enabled(false)
+                .enabled(true)

```

Comment:
This should be false, since newly registered user can't log in until admin has approved his registration request